supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc

```diff
@@ -125,14 +125,31 @@ For example, instead of "Red Hat recommends using X package because", write "Use
 [discrete]
 [[red-hat-container-catalog]]
 ==== Red Hat Container Catalog (noun)
-*Description*: "Red Hat Container Catalog" is Red Hat's hosted registry for Enterprise-ready containers located at link:https://access.redhat.com/containers[access.redhat.com/containers]. Red Hat Container Catalog is a trusted source for secure, certified, and up-to-date container images. Write this name in full the first time that you use it in a document.
+*Description*: "Red Hat Container Catalog" was Red Hat's hosted registry for enterprise-ready containers located at link:https://access.redhat.com/containers[access.redhat.com/containers].
 
-*Use it*: yes
+The Red Hat Container Catalog no longer exists; it has become part of the Red Hat Ecosystem Catalog, which holds not only information on container images, but also information on certified software, hardware, and cloud service providers. The old link:https://access.redhat.com/containers[Red Hat Container Catalog] link redirects to the link:https://catalog.redhat.com/software/containers/explore[Container images] section of the Red Hat Ecosystem Catalog.
+
+*Use it*: no
 
 *Incorrect forms*:
 
 *See also*: xref:container-registry[container registry], xref:openshift-container-registry[OpenShift Container Registry]
 
+[discrete]
+[[red-hat-ecosystem-catalog]]
+==== Red Hat Ecosystem Catalog (noun)
+*Description*: The "Red Hat Ecosystem Catalog" is the official source for discovering and learning more about the Red Hat Certified Technology Ecosystem and certified third-party products and services. The Red Hat Ecosystem Catalog is a repository for all certified partner products (software, hardware, or a public cloud provider image) that run on-top, inside, or underneath Red Hat software (Red Hat Enterprise Linux, OpenShift Container Platform, Red Hat OpenStack Platform, Ansible, and so on).
```

Comment:
I don't think on-top should be hyphenated.

Reply:
Maybe this could work?
`..that run on, in, or under Red Hat software..`
Don't think there's any difference in meaning between 'on' and 'on top' or 'under' and 'underneath' .. seems a bit fluffy and redundant if 'in', 'on', and 'under' do the same job?

Also could reduce the parentheses:
The Red Hat Ecosystem Catalog is a repository for all certified partner products, software, hardware, or a public cloud provider image, that run on, in, or under Red Hat software, for example, Red Hat Enterprise Linux, OpenShift Container Platform, Red Hat OpenStack Platform, and Ansible.

If you go with 'for example' the suggestion is that the list is non exhaustive so you wouldn't need 'and so on'.

Reply:
I like it.

Reply:
Good suggestions, adjusting!